### PR TITLE
major update of `immuno_graphical` and `cor_graphical`; update specs

### PR DESCRIPTION
**Pull request by KenLi93**

### CODE REVIEW

File: cor_graphical/code/CoR_assay_graphics.R [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#-----------------------------------------------
+v#-----------------------------------------------
```
Comment (nhejazi) [inline cor_graphical/code/CoR_assay_graphics.R:1]: this looks like a syntax error

File: _common.R [hunk 2]
```diff
@@ -92,7 +92,7 @@ Bstratum.labels <- c(
   "Age < 65, Not at risk"
 )
 
-llods <- c(bindSpike=20, bindRBD=20, pseudoneutid50=10, pseudoneutid80=10) 
+llods <- c(bindSpike=20, bindRBD=20, pseudoneutid50=10, pseudoneutid80=10)
```
Comment (nhejazi) [inline _common.R:95]: please use the LLODs already included in `_common.R`, https://github.com/CoVPN/correlates_reporting/blob/master/_common.R#L95

File: immuno_graphical/.gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 data_clean/*.Rdata
 data_clean/*.rds
+data_clean/practice_data.csv
```
Comment (nhejazi) [inline immuno_graphical/.gitignore:3]: this should not be where the `practice_data.csv` lives. it is already `.gitignore`d at the top level `data_clean` directory.

**Review by nhejazi — CHANGES_REQUESTED** [review]
In addition to the comments below, the file `cor_graphical/data_clean/practice_data.csv` should not exist. The naming suggests that it is a copy of the same file in the top level `data_clean` directory (i.e., in `correlates_reporting/data_clean/`, as created by `make_dat_proc.R`). If this is the case, directly use that file instead of copying it.

**Comment by nhejazi** [comment]
@KenLi93, see the PR review above for the necessary changes. I would advise waiting until we have a successful automated build prior to making any changes since we will not be merging PRs that fail the automated build checks reported above.

**Comment by KenLi93** [comment]
Thank you for the comments!

**Pull request closed without merging**